rsts/getting_started.rst

```diff
@@ -44,15 +44,15 @@ Make sure you have `Git <https://git-scm.com/>`__, and `Python <https://www.pyth
 
 Start a new project / repository
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Install Flyte's Python SDK — `Flytekit <https://pypi.org/project/flytekit/>`__ (recommended using a  `virtual environment <https://docs.python.org/3/library/venv.html>`__) and clone the `flytekit-python-template <https://github.com/flyteorg/flytekit-python-template>`__ repo.
+Install Flyte's Python SDK — `Flytekit <https://pypi.org/project/flytekit/>`__ (recommended using a  `virtual environment <https://docs.python.org/3/library/venv.html>`__), run a command to initialize your project, that command will prompt you for the name of directory where your example workflows are going to live. The ``init`` command comes with a help menu that can be shown by running ``pyflyte init --help``. For this guide we're going to use ``myflyteapp`` in the initial invocation.
```

Comment:
I dont think we need the extra text. That is the beauty of CLI's?

Reply:
I guess I was trying to hand-hold too much. On a second read I agree with you, let me remove that part.